Kapacitor: clarify behavior of StateCountNode and StateCountDuration (#2572)
influxdata/docs-v2: content/kapacitor/v1.5/nodes/state_duration_node.md

---

The `stateDuration` node computes the duration of a given state.
The state is defined via a lambda expression.

> For each consecutive point for which the expression evaluates as `true`,
> the state duration will be incremented by the duration between points.

Reviewer: ```suggestion
For each consecutive point the lambda expression evaluates as `true`,
the state duration is incremented by the duration between points.
```
